src/renderer/hooks/useDeeplinking.js

```diff
@@ -165,6 +166,10 @@ export function useDeepLinkHandler() {
           break;
         }
 
+        case "settings":
+          navigate(`/settings/${splitedPath[1]}`);
```

Comment:
what happens if splittedPath[1] === undefined?
i think it will redirect to `/settings/undefined` try do a sanity check here in the url maybe,
Also i think would be great tp filter out the splittedPath[1] and limit it to a subset defined list of values like
```javascript
const listOfSettingsPaths = ['account', 'general', 'experimental', ...];

navigate(`/settings${ listOfSettingsPaths.includes(splitedPath[1]) ? `/${splitedPath[1]}`  : ""}`);
```

Reply:
good catch, indeed

Reply:
It does redirect to /settings/undefined, thus redirecting to /settings (not clean indeed)
I have put a sanity/whitelist check (using a switch) cause i needed to change /settings/display link to /settings/general anyway